ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/apitesting/LedgerContext.scala

```diff
@@ -94,6 +92,11 @@ trait LedgerContext {
   def partyManagementService: PartyManagementService
   def packageManagementService: PackageManagementService
 
+  /** Default forParty implementation */
+  final def default: LedgerContext = forParty(None)
+  final def forParty(party: Party): LedgerContext = forParty(Some(party))
+  def forParty(party: Option[Party]): LedgerContext = this
```

Comment:
Overloading methods in Scala can be a problem. I'd rather recommend giving them unique names.

Reply:
Indeed: https://stackoverflow.com/questions/2510108/why-avoid-method-overloading :open_mouth:

Reply:
Done